library/tinkerpop/src/main/java/uk/gov/gchq/gaffer/tinkerpop/GafferPopGraph.java

```diff
@@ -518,7 +517,8 @@ public <T> T execute(final OperationChain<T> opChain) {
         try {
             LOGGER.info("GafferPop operation chain called: " + opChain.toString());
             return graph.execute(opChain, user);
-        } catch (final OperationException e) {
+        } catch (final Exception e) {
+            LOGGER.error("Operation chain failed: " + e.getMessage(), e);
             throw new RuntimeException(e);
```

Comment:
We can add a message to this type of exception.
```suggestion
            throw new RuntimeException("Failed to execute GafferPop operation chain", e);
```